### PR TITLE
add notes for n25380

### PR DESCRIPTION
**Pull request by glozow**

### CODE REVIEW

File: _posts/2022-07-13-#25380.md [hunk 1]
```diff
@@ -7,18 +7,73 @@ authors: [darosior]
...
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- Fees for the wallet are calculated using a fee estimator attached to the node's mempool,
+  [`CBlockPolicyEstimator`](https://github.com/bitcoin/bitcoin/blob/994573726f81afca5343e2a8d82309e697e61f90/src/policy/fees.h#L132).
+
+    - At a high level, it records the feerate and time at which transactions enter the mempool and
+      how many blocks later they confirm. Later, it uses this information ot provide the opposite
```
Comment (stickies-v) [inline _posts/2022-07-13-#25380.md:19]:
```suggestion
      how many blocks later they confirm. Later, it uses this information to provide the opposite
```

File: _posts/2022-07-13-#25380.md [hunk 2]
```diff
@@ -7,18 +7,73 @@ authors: [darosior]
...
+
+    (a) What information is recorded for new transactions entering the mempool? (Hint: what does
+        [`mapMemPoolTxs`](https://github.com/bitcoin/bitcoin/blob/194710d8ff398838e4e5bb87b56e19ebed1d6c52/src/policy/fees.h#L258) store?)
+
+    (b) What information is recorded for transactions in blocks? (Hint: see [`processBlockTx`](https://github.com/bitcoin/bitcoin/blob/194710d8ff398838e4e5bb87b56e19ebed1d6c52/src/policy/fees.cpp#L596))
+
+    (c) Does the fee estimator track any transactions that are not in the mempool, were removed, or
+        were mined but not in the mempool beforehand?
+
+1. How does the PR detect which transactions to drop? Do you agree with this approach?
```
Comment (stickies-v) [inline _posts/2022-07-13-#25380.md:70]: nit, but:
```suggestion
1. In this PR, how do we detect which transactions to drop? Do you agree with this approach?
```

**Review by stickies-v — APPROVED** [review]
Two minor comments, but otherwise ACK 8911af90169b460cbae21b66dde65528b2f79ce2

**Comment by glozow** [comment]
Addressed both comments, thanks @stickies-v! :)

**Pull request merged**